easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.0.2-gompi-2019b-OpenBLAS-0.3.6.eb

```diff
@@ -0,0 +1,22 @@
+name = 'ScaLAPACK'
+version = '2.0.2'
+
+homepage = 'http://www.netlib.org/scalapack/'
+description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
+ redesigned for distributed memory MIMD parallel computers."""
+
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = [SOURCELOWER_TGZ]
+checksums = ['0c74aeae690fe5ee4db7926f49c5d0bb69ce09eea75beb915e00bba07530395c']
+
+local_blaslib = 'OpenBLAS'
+local_blasver = '0.3.6'
+
+versionsuffix = "-%s-%s" % (local_blaslib, local_blasver)
```

Comment:
Maybe remove this `versionsuffix` too since we have only one version of `OpenBLAS`?

Reply:
fixed in 29b9691